<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportOptionsWidget</class>
 <widget class="QWidget" name="ExportOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="pdfExportGroup">
     <property name="title">
      <string>PDF Export</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="noDpiLimitCB">
        <property name="text">
         <string>No DPI limit (use Output resolution)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="maxDpiLayout">
        <item>
         <widget class="QLabel" name="maxDpiLabel">
          <property name="text">
           <string>Max Image DPI:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxDpiSpinBox">
          <property name="minimum">
           <number>72</number>
          </property>
          <property name="maximum">
           <number>1200</number>
          </property>
          <property name="value">
           <number>400</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="maxDpiSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="compressGrayscaleCB">
        <property name="text">
         <string>Compress grayscale (JPEG)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="qualityLayout">
        <item>
         <widget class="QLabel" name="qualityLabel">
          <property name="text">
           <string>JPEG Quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="qualityCombo"/>
        </item>
        <item>
         <spacer name="qualitySpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="exportToPdfBtn">
        <property name="text">
         <string>Export to PDF...</string>
        </property>
        <property name="icon">
         <iconset theme="document-save-as"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
